src/pages/ExamsPage/data/reducer.test.js

```diff
@@ -0,0 +1,44 @@
+import {
+  initialState,
+  reducer,
+} from './reducer';
+
+describe('ExamsPage reducer', () => {
+  it('returns the initial state', () => {
+    expect(reducer(undefined, {})).toEqual(initialState);
+  });
+  describe('actions', () => {
+    describe('loadExams', () => {
+      it('sets the examsList', () => {
+        const action = {
+          type: 'exams/loadExams',
+          payload: [
+            {
+              id: 1,
+              exam_name: 'Exam 1',
+              exam_type: 'timed',
+            },
+            {
+              id: 2,
```

Comment:
Question: so the exam list has limited exam info, but I'm guessing if we needed more info, we would call into edx-exams? And would this be using hooks?

Reply:
good question, we're only pulling the fields we need out of the response in the reducer/action below. If we want other fields from the api we can just map them to state inside loadExams. For example exam_type isn't included in state right now but is in the response data.

https://github.com/edx/frontend-app-exams-dashboard/blob/78bdbff9e60b80e42488e8b247f9307eb6dc7ccf/src/pages/ExamsPage/data/reducer.js#L14